design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
```

Comment:
Why not SKU?

Reply:
SKU is not a unique guaranteed identifier for the catalog.
Along with this product exposes ID as a part of product type.